src/javascript/app/pages/trade/common_independent.js

```diff
@@ -133,12 +133,6 @@ const getSelectedOption = ($selector) => {
     return $selected_option;
 };
 
-const showAssetOpenHours = ($selector) => {
-    const $selected_option = $selector ? getSelectedOption($selector) : '';
-    const sessions         = $selected_option ? $selected_option.attr('data-sessions') : '';
```

Comment:
seems like we can also remove `data-sessions`?

Reply:
we can't because `data-sessions` is also used to create forward starting dates in the dropdown

Reply:
where is that? I only see its usage in starttime.js where it's being set, not checked